proof/crefine/ARM/Ipc_C.thy

```diff
@@ -2349,7 +2349,7 @@ lemma transferCapsLoop_ccorres:
                  \<inter> \<lbrace>\<acute>destSlot = (if slots = [] then NULL else cte_Ptr (hd slots))
                                 \<and> length slots \<le> 1 \<and> slots \<noteq> [0]\<rbrace>)"
   defines "is_the_ep \<equiv> \<lambda>cap. isEndpointCap cap \<and> ep \<noteq> None \<and> capEPPtr cap = the ep"
-  defines "stable \<equiv> \<lambda>scap excap. excap \<noteq> scap \<longrightarrow> excap = maskedAsFull scap scap"
+  defines "stable_masked \<equiv> \<lambda>scap excap. excap \<noteq> scap \<longrightarrow> excap = maskedAsFull scap scap"
```

Comment:
There is now an alternative constant called stable that is now visible in HOL, so I just changed the name of the local definition in this lemma. Suggestions welcome for alternative names.

Reply:
I'm fine with `stable_masked`, it describes (some of) what is going on.